src/js/views/patients/schedule/schedule-list.scss

```diff
@@ -79,7 +79,11 @@
     background: $greyed;
   }
 
-  &:hover:not(.is-selected) {
+  &.is-reduced {
+    cursor: default;
```

Comment:
I switched the cursor to a `default` version here. Since the row is not clickable, I figured it wouldn't make sense to use the pointer cursor.

Same change was made for the patient name.